python/notebooks/00-getting-started.ipynb

```diff
@@ -49,17 +49,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
```

Comment:
please clear all outputs from Jupyter notebooks. Changes to the notebooks need to be committed without any output.